pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1415,7 +1415,8 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 		CPUs:      cpuCount,
 	}
 	// set the maximum number of sockets here to allow hot-plug CPUs
-	if vmiCPU := vmi.Spec.Domain.CPU; vmiCPU != nil && vmiCPU.MaxSockets != 0 {
+	// CPU hotplugging for ARM64 is currently unsupported
+	if vmiCPU := vmi.Spec.Domain.CPU; vmiCPU != nil && vmiCPU.MaxSockets != 0 && !isARM64(c.Architecture) {
```

Comment:
I would make this explicit by asking, is this x86?

Reply:
I'm relatively sure that s390x for example also supports CPU hotplug[1], should we explicitly limit this feature to x86?
https://docs.kernel.org/core-api/cpu_hotplug.html